.github/workflows/test.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        goVer: ['1.16', '1.17', '1.18']
+        goVer: ['1.18', '1.19', '1.20', '1.21']
```

Comment:
why remove 16 and 17?

Reply:
https://github.com/vesoft-inc/nebula-go/pull/292/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R3

so we should set the minimal support version as `1.16`?